testng-core/src/main/java/org/testng/internal/ExitCode.java

```diff
@@ -19,7 +19,7 @@ public class ExitCode {
   public static final int HAS_NO_TEST = 8;
   private static final int FAILED_WITHIN_SUCCESS = 4;
   public static final int SKIPPED = 2;
-  private static final int FAILED = 1;
+  public static final int FAILED = 1;
```

Comment:
Add comment: visible for tests

Reply:
This is a constant. Its not going to be altered. The comment is just going to be redundant IMO

Reply:
Ok